### DISCUSSION TITLE
LDM Notes for August 21st, 2024

### DISCUSSION THREAD

**Discussion by 333fred** [dotnet/csharplang#8378] [LDM Notes]
https://github.com/dotnet/csharplang/blob/main/meetings/2024/LDM-2024-08-21.md

### Agenda

- Better conversion from collection expression
- `field` keyword nullability

**Comment by jnm2** [comment]
> or "getters that have *more* nullability warnings than when `field` starts as null";

Going forward, I think we should say ***new*** nullability warnings rather than ***more*** nullability warnings. I think this achieves that clarity that we noticed was lacking in the word 'more.' (We're not comparing total counts.) @CyrusNajmabadi @RikkiGibson 

Basically, `var isNullResilientGetter = !diagsWithFieldMaybeNull.Except(diagsWithFieldNotNull).Any()`.
